PSI_Project/Models/BaseEntity.cs

```diff
@@ -1,10 +1,10 @@
 ﻿namespace PSI_Project.Models;
 
 // 1: using our own class
-public class BaseEntity
+public  class BaseEntity
 {
     public string Id { get; init; } // 2: property usage in class (get, init, set)
-    public string Name { get; set; }
+    public string Name { get; set; } // do we really need Name after all?
```

Comment:
I don't think that we do. Some classes that extend BaseEntity class definitely don't need name. So I think we can get rid of it from here.

Reply:
For some of our classes like: subject, topic and conspectus, it was ok, but now that we will have more classes, some of them won't need the name, so we can get rid of it probably